Automattic/vip-go-mu-plugins
# CANTINA-913: Match protocol in CORS header

## Description

Only allow HTTPS origins on VIP

## Changelog Description

### Filter added: vip_only_https_origins

Only allow HTTPS origins on VIP

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:


## Pre-deploy checklist


## Steps to Test
1) `curl 'https://example.net/wp-admin/admin-ajax.php?action=heartbeat' -H 'Origin: http://example.net' -I` and expect to see `access-control-allow-origin` returned
2) Apply patch
3) Do step 1 again and expect to not see `access-control-allow-origin` returned
